docs/manual-EN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -150,6 +151,17 @@ Check the logs of the storaged service. If you find the "wrong cluster" error me
 
 Use the command `curl http://ip:port/status` to obtain the git_info_sha, the commitID of the binary package.
 
+### Modifying the Configuration File Does not Take Effect
+
+**Nebula Graph** uses the following two methods obtaining configurations:
+
+1. From the configuration files (You need to modify the files then restart the services);
+2. From the Meta. Set via CLI and persists in Meta service. Please refer to the [Configs Syntax](../../3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md) for details.
+
+Modifying the configuration file does not take effect happens because **Nebula Graph** gets configuration in the second method (from meta) by default. If you want to use the first way, please add the `local_config=true` option in flag files `metad.conf`, `storaged.conf`, `graphd.conf` (flag files directory is `/usr/local/nebula/etc/`) respectively.
```

Comment:
Modifying the configuration file does not take effect because **Nebula Graph** gets configuration in the second method (from meta) by default.